app/views/proposals/details/_request_details.html.haml

```diff
@@ -7,4 +7,4 @@
             Request Details
         .content-content.column#view-request-details
           .row
-            = render @current_user.client_slug + '/request_details'
+            = render client_specific_partial(@proposal.requester, '/request_details')
```

Comment:
Should this `client_specific_partial()` method be used elsewhere above (like in the change made to `_request_details.html.haml`, when it loads a client-specific partial)? Or do we have something like it already elsewhere?

Also, the current implementation of `client_specific_partial()` takes a `User` as the first argument, which is kind of strange. Can we tweak it just to take a client slug?